<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QWidget" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>448</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Droid Arabic Kufi</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>تسجيل الدخول</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../pathIcon.qrc">
    <normaloff>:/images/login.png</normaloff>:/images/login.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>0.980000000000000</double>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <property name="locale">
   <locale language="Arabic" country="Jordan"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>40</y>
     <width>271</width>
     <height>211</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../pathIcon.qrc">:/images/login.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="linEditUser">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>270</y>
     <width>281</width>
     <height>29</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>20</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>اسم المستخدم</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPassword">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>320</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="maxLength">
    <number>25</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>كلمة المرور</string>
   </property>
  </widget>
  <widget class="QLabel" name="labReturnPass">
   <property name="geometry">
    <rect>
     <x>84</x>
     <y>360</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>هل نسية كلمة المرور ؟</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLogin">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>398</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>دخول</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>177</x>
     <y>398</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>الغاء</string>
   </property>
  </widget>
  <widget class="QLabel" name="labLoginUser">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>308</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>تسجيل الدخول</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>linEditUser</zorder>
  <zorder>lineEditPassword</zorder>
  <zorder>btnLogin</zorder>
  <zorder>btnClose</zorder>
  <zorder>label</zorder>
  <zorder>labReturnPass</zorder>
  <zorder>labLoginUser</zorder>
 </widget>
 <tabstops>
  <tabstop>btnClose</tabstop>
  <tabstop>linEditUser</tabstop>
  <tabstop>lineEditPassword</tabstop>
  <tabstop>btnLogin</tabstop>
 </tabstops>
 <resources>
  <include location="../pathIcon.qrc"/>
 </resources>
 <connections/>
</ui>
